examples/rust/msm/src/main.rs

```diff
@@ -74,40 +74,39 @@ fn main() {
         let size = 1 << log_size;
         println!("---------------------- MSM size 2^{}={} ------------------------", log_size, size);
         // Setting Bn254 points and scalars
-        let points = &upper_points[..size];
-        let scalars = &upper_scalars[..size];
+        let points = HostOrDeviceSlice::Host(upper_points[..size].to_vec());
```

Comment:
@svpolonsky add a G2 example to here as well

Reply:
I will implement this change in  `update-examples-for-new-API-2`